### DISCUSSION TITLE
The hidden sidebar doesn't fully expand on my machine, where can I change it?

### DISCUSSION THREAD

**Discussion by gitttyyy** [soulhotel/FF-ULTIMA#17] [Q&A] [answered]
I'm not very familiar with the front end🐶
![image](https://github.com/soulhotel/FF-ULTIMA/assets/154318356/5b2f03ad-add3-4ca4-bbec-4601c42e1062)

**Comment by gitttyyy** [comment]
OK,I find it🐶

**Comment by Kaedriz** [comment, marked as answer]
For anyone else wondering, you need to change `@import url(theme/all-global-positioning-s.css);` in `userChrome.css`. Change `-s.css` part of that to `-l.css`.

I'm wondering if this maybe should be default behavior, as in my opinion is much more useful, when paired with autohide.
